### PR TITLE
Added logging configuration example to readme, refactored logging using

### PR DESCRIPTION
**Pull request by Miraak777**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,4 +92,40 @@ server.stop()
...
 
 ## Utils
-...
+### Logging
+You can configurate logs as you want by config.
+
+Example:
+
+```py
+from sc_kpm.constants import MAIN_LOGGER
```
Comment (AlexandrZagorskiy) [inline README.md:101]: change name from MAIN_LOGGER to LOGGER_NAME, we have no another loggers. Also I want to import LOGGER_NAME directly from the sc_kpm.

File: README.md [hunk 2]
```diff
@@ -92,4 +92,40 @@ server.stop()
...
+            }
+        },
+        loggers={
+            MAIN_LOGGER: {"handlers": ["your_custom_handler", "your_another_custom_handler"],
+                          "level": logging.DEBUG, }
+        },
+    )
+)
+```
+### Note:
```
Comment (AlexandrZagorskiy) [inline README.md:129]: remove it, write note at the same subsection with logging example

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: src/sc_kpm/constants.py [hunk 3]
```diff
@@ -0,0 +1 @@
+MAIN_LOGGER = "main_logger"
```
Comment (AlexandrZagorskiy) [inline src/sc_kpm/constants.py:1]: rename value of the LOGGER_NAME to the "py-sc-kpm"

**Review by AlexandrZagorskiy — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Miraak777) [reply README.md:101]: actually, you can import it directly from sc_kpm

Comment (FalseR20) [reply README.md:101]: no, I think better to use full import. it can brake import order

**Review by FalseR20 — APPROVED** [review]

**Review by AlexandrZagorskiy — APPROVED** [review]

**Pull request merged**